uplink/src/base/actions.rs

```diff
@@ -105,7 +114,7 @@ impl ActionResponse {
 
 impl Point for ActionResponse {
     fn stream_name(&self) -> &str {
```

Comment:
An `ActionResponse` with `action_name == None` will go to the `action_status` stream i.e. topic: `/tenants/{tenant_id}/devices/{device_id}/action/status`
Whereas those with `action_name == Some(stream_name)` will go onto their streams and have the topic: `/tenants/{tenant_id}/devices/{device_id}/action_status/{stream_name}`